src/cookie.rs

```diff
@@ -0,0 +1,133 @@
+use crate::deref;
+use cookie::Cookie as CookieRs;
+use cookie::CookieBuilder;
+
+/// Represents the cookies, cookies are stored
+/// with the name and values. By default you have
+/// yourself a `Cookies` field in `Response`.
+///
+/// Cookies are behind the feature `cookies` which is included
+/// in the default one
+///
+/// # Example
+///
+/// ```
+/// use octane::prelude::*;
+///
+/// let mut app = Octane::new();
+/// app.get(
+///     "/",
+///     route!(|req, res| {
+///         res.cookie(octane::cookie::Cookie::new("name", "value")).send("Cookie has been set!");
+///         let sent_cookie = &req.request.cookies;
+///         // Do stuff with cookies
+///         Flow::Stop
+///     }),
+/// );
+/// ```
+#[derive(Debug, Clone, PartialEq)]
+pub struct Cookie<'a> {
+    cookie: CookieRs<'a>,
+}
+
+impl Cookie<'_> {
+    /// Creates a new `Cookie` instance with the given `name` and `value`.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use octane::prelude::*;
+    ///
+    /// let cookie = octane::cookie::Cookie::new("name", "value");
+    /// println!("{:?}", cookie);
+    /// ```
+    pub fn new<'a>(name: &'a str, value: &'a str) -> Cookie<'a> {
+        Cookie {
+            cookie: CookieRs::new(name, value),
+        }
+    }
+    /// Creates a new `cookie` instance from a `cookie` in the `cookie-rs` library.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use octane::prelude::*;
+    /// use cookie::Cookie as CookieRs;
+    ///
+    /// let cookie_rs_cookie = CookieRs::new("name", "value");
+    /// let octane_cookie = octane::cookie::Cookie::from(cookie_rs_cookie);
+    /// println!("{:?}", octane_cookie);
+    /// ```
+    pub fn from<'a>(cookie: CookieRs<'a>) -> Cookie<'a> {
```

Comment:
Use the from trait here instead of an function https://doc.rust-lang.org/std/convert/trait.From.html

Reply:
Fixed.